### ISSUE TITLE
Previous versions

### ISSUE THREAD

**Issue by jamiecounsell** [anewandrew/Advanced-Journal-Challenge#1]
One of the reasons we use GIT is so we can see old versions in the same repository. Ideally, you would only have one journal-challenge repository, and make many commits as you improve it. If I need to see an older version, I can always look into the project's history! :dancer:

**Comment by anewandrew** [comment]
Yeah, I realize that. One is an older journal challenge for object practice I think, the other is the actual web journal/blog project. They're both two different projects.

**Comment by jamiecounsell** [comment]
The objects from the first Journal challenge are supposed to be a start for the more advanced journal challenge, so we can just improve on older code. No worries for now, but something to keep in mind for the future! The other option later on will be to create a branch for different "versions" of a repository.
